routes/intallatievergadering.ts

```diff
@@ -333,4 +349,272 @@ async function movePersons(installatievergaderingId: string) {
   await updateSudo(sparql);
 }
 
+async function copyCommuneMandatarisInstancesToOCMW(
```

Comment:
If you want to specify from where to where you copy, I would use municipality, that is the more often used translation for "gemeente".